### ISSUE TITLE
Cannula insertion timeout

### ISSUE THREAD

**Issue by bluddydesign** [LoopKit/Loop#1061]
Hello Loop Users,

I'm new to the loop and I'm having trouble getting set up. Everything looks good on the app (riley link, omnipod priming, etc) until the cannula insertion and the app just closes out. Please take a look at my log file attached if you can.

I’m thinking I should reinstall the app on my iphone from my computer but please let me know if there is something I should be looking for either on the app end or the reinstallation end? Thank you for taking a look!

https://loop.zulipchat.com/user_uploads/8196/R3avDQ8q88yWNuc0fhF7DG2I/Loop-Report-2019-08-14-143931-0700.md

**Phone**
 - Hardware: iPhone 8
 - OS Version: iOS 12.4

**Loop Version**
 - Version Number: V1.10.0DEV

**CGM**
 - Device: Dexcom G6
 - Manager app: Dexcom App

**Pump**
 - Manufacturer: Omnipod

**Comment by ps2** [comment]
Looking at your logs, your Loop is doing something I wouldn't expect to see from a normal build of the dev branch.  You didn't happen to make a code customization to enable expiration alerts did you? There is a section of code that can be uncommented to do that, and that section is right around where your loop seems to be taking a wrong turn in the pod setup process.

**Comment by bluddydesign** [comment]
Hi ps2, Thanks for taking a look at my logs! As far as I know I didn't make any customization to the code and I don't really code in the first place... Is there something I should look for in the code to delete or modify to stop this from happening?

**Comment by ps2** [comment]
If this happens again, please start loop via Xcode and keep your phone connected. When it crashes, take a screenshot of the Xcode screen. That will help us see where it’s crashong.

**Comment by Kdisimone** [comment]
@bluddydesign (video if you are unsure how to get the Xcode info pete’s Talking about...https://youtu.be/Ac4MguvUO7M for crashes, the logs will stop, so you’d just capture the screen when the crash comes up and get all the info then. The key part is you have to build to get Xcode connected to your loop beforehand.

**Comment by bluddydesign** [comment]
Thank you kindly ps2 and Kdisimone,

I was able to successfully get the Loop to insert the Cannula when I reinstalled the Loop & kept my phone plugged into Xcode! I was able give myself a tiny test bolus with the phone plugged in and also unplugged from my computer! Not sure if tethering my iphone was the issue? Hoping to be able to instal the omnipod/cannula untethered in 3 days. Please advise if that might be an issue.

Here's my gist: https://gist.github.com/bluddydesign/8f260f2fd5c880ef1e806b3248aa0828

Thanks so much. Please let me know if it looks like I'm good now or if tethering might be an issue.

Best Regards,
Ben

**Comment by bearonatrail** [comment]
I am having the same issue on our first time setting up the link. Any suggestions or explanation of what worked to fix it?

**Comment by TanyaSpencer** [comment]
I am having the same issues with my daughter.

**Comment by ASHMaple28** [comment]
Did anyone figure out what the problem might be here? We had the same problem the first time we tried to set up Loop for my son. I posted about it in this same thread in the FB group but I thought I’d ask here too. I know some of you are busy in Madrid 😊 but maybe someone else has a solution I’m missing. As far as I can tell I don’t have any of the problems mentioned elsewhere (like backwards basal ranges). Going to try again this afternoon with the phone plugged into the computer and Xcode running, but we’ve already lost one pod and I hate to waste another if there’s a simple fix I’m missing. Thank you!
